### PR TITLE
Fix incorrect version in openshift-oc-release-4.18.yaml

### PR DESCRIPTION
**Pull request by ashwindasr**
The 4.18 file had references to 4.17

### CODE REVIEW

File: ci-operator/config/openshift/oc/openshift-oc-release-4.18.yaml [hunk 1]
```diff
@@ -33,35 +33,35 @@ build_root:
...
-    ocp_4.17_base-rhel9:
+    ocp_4.18_base-rhel9:
       as:
-      - registry.ci.openshift.org/ocp/4.17:base-rhel9
-    ocp_builder_rhel-9-golang-1.22-openshift-4.17:
+      - registry.ci.openshift.org/ocp/4.18:base-rhel9
+    ocp_builder_rhel-9-golang-1.22-openshift-4.18:
       as:
-      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.17
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.18
```
Comment (thegreyd) [inline ci-operator/config/openshift/oc/openshift-oc-release-4.18.yaml:50]: these pullspecs aren't populated yet

Comment (thegreyd) [reply ci-operator/config/openshift/oc/openshift-oc-release-4.18.yaml:50]: these are live now, marking this resolved

**Comment by ardaguclu** [comment]
Thank you for fixing this.

**Comment by ardaguclu** [comment]
It seems that this commit https://github.com/openshift/release/commit/80b23353d87bed2a4b2acaf3f4cd70511f75e590 has merged before than this and causing conflicts. Opened this https://github.com/openshift/release/pull/56386 to fix.

**Pull request merged**